### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by akroon3r**
Including steps to create a consumable zipped build for release.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,30 +2,22 @@
...
-## Running Locally
+## Release packaging
 
-In order to use these shared react components locally, run:
+Run the following commands locally after a new component has been added:
 
-```bash
-yarn prepare
-```
+`bash yarn install`
```
Comment (sdevalapurkar-bcgov) [inline README.md:9]: this will show up as `bash yarn install` which we dont want
should be:
```bash
yarn install
```
same below

File: README.md [hunk 2]
```diff
@@ -2,30 +2,22 @@
 
 Shared reusable react BCGov themed components.
 
-## Running Locally
+## Release packaging
 
-In order to use these shared react components locally, run:
+Run the following commands locally after a new component has been added:
```
Comment (sdevalapurkar-bcgov) [inline README.md:7]:
```suggestion
Run the following commands locally when ready to release:
```

File: README.md [hunk 3]
```diff
@@ -2,30 +2,22 @@
...
-```
+`bash yarn install`
 
-```bash
-npm install -g linklocal
-```
+`bash npm pack`
 
-from the `shared-components` directory.
+Running the pack command will create a *shared-components-X.X.X.tgz* file in the root level of the projectthat contains the components ready for consumption by any project.
```
Comment (sdevalapurkar-bcgov) [inline README.md:13]:
```suggestion
Running the pack command will create a *shared-components-X.X.X.tgz* file in the root level of the project that contains the components ready for consumption by any project.
```

File: README.md [hunk 4]
```diff
@@ -2,30 +2,22 @@
...
 
-Then, to include the component dependency and include it in your react app:
+In order for projects to consume a new version, create a new release on this repository with the updated semantic version. Once the release is created, edit the release and upload the generated *shared-components-X.X.X.tgz* file to the release. Once this is complete, update your projects package.json to the following:
```
Comment (sdevalapurkar-bcgov) [inline README.md:15]:
```suggestion
In order for projects to consume a new version, create a new release on this repository with the updated semantic version. Once the release is created, edit the release and upload the generated *shared-components-X.X.X.tgz* file to the release. Once this is complete, update your project's package.json file to the following:
```

File: README.md (hunk 1 above)
Comment (sdevalapurkar-bcgov) [reply README.md:9]: so use
```suggestion
```bash
yarn install
```
```

File: README.md [hunk 5]
```diff
@@ -2,30 +2,28 @@
...
 
 ```bash
-yarn prepare
+yarn install
 ```
 
-```bash
-npm install -g linklocal
+```
+bash npm pack
```
Comment (sdevalapurkar-bcgov) [inline README.md:14]: no bash here

Comment (sdevalapurkar-bcgov) [reply README.md:14]: bash should be beside the ```

Comment (sdevalapurkar-bcgov) [reply README.md:14]: looked like it was correct before :)

Comment (akroon3r) [reply README.md:14]: whoopsie

**Comment by akroon3r** [comment]
:shipit:

**Pull request merged**